crates/bootstrap_client/src/lib.rs

```diff
@@ -0,0 +1,49 @@
+//! A client for the Kitsune2 bootstrap server.
+
+#![deny(missing_docs)]
+
+use kitsune2_api::{AgentInfoSigned, DynVerifier, K2Error, K2Result, SpaceId};
+use std::sync::Arc;
+
+/// Send the agent info, for the given space, to the bootstrap server.
+pub fn put(
+    server_url: Arc<str>,
+    agent_info: Arc<AgentInfoSigned>,
```

Comment:
```suggestion
    server_url: &str,
    agent_info: &AgentInfoSigned,
```

I think this will be a more generally useful api... to me it makes sense to have the complexity of moving the data into the spawn_blocking closure be a concern at that level when implementing the spawn_blocking.

Reply:
Done: https://github.com/holochain/kitsune2/pull/129/commits/531a406292a63bb0d27852900ca937266eec7d21